src/components/pets/pets.css

```diff
@@ -49,3 +49,7 @@
   align-self: center;
   border-radius: 4;
 }
+.petList__container.card{
```

Comment:
add a space between `.petList__container .card`

Reply:
You told that className="card" not required so here that is src/components/pets/pets.css I have to remove this card class and use inline css in that Card tag right ?

Reply:
the bootstrap css already has a selector named `card` we are just modifying it in the `.petList__container`. so we could just do `.petList__container .card` in order to modify the bootstrap css card

Reply:
Ok got it 👍